src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -72,33 +63,25 @@ interface Props {
   isContributorsSubsection: boolean;
 }
 
-// const UserAvatar = HookedUserAvatar({ fetchUser: true });
-
-// const MAX_AVATARS = 12;
+const MAX_AVATARS = 12;
 
 const MembersSubsection = ({
   colony: { name, watchers },
   // members,
```

Comment:
Is that still needed?

Reply:
Leaving this in place as it will be used when the members data is eventually passed in from the ColonyMembersWidget